### PR TITLE
Added in progress status to job log

### LINKED ISSUE #250: Add "In Progress" status to job logs
The "In Progress" status should be added to job logs as in the backend repository

### LINKED ISSUE #252: Create decorator for functions using `job_log`
A decorator could be used instead of calling job log several times in each function. The decorator would set the status to in progress before calling the wrapped function, and then depending on the return values (maybe an exception would work for failure too) of the wrapped function success/fail with a provided message string.

### PR DESCRIPTION
**Pull request by Fagtoy**
Added "In Progress" status to job logs, created a separate job log enum with all the statuses, and added the ```job_log``` function call to other scripts.

resolves #250
resolves #252

### CODE REVIEW

**Review by richardzilincikPantheon — APPROVED** [review]
Base your branches on `develop` whenever possible.

**Comment by Fagtoy** [comment]
> Base your branches on `develop` whenever possible.

I do but just didn't want to resolve possible conflicts due to moving code out of the ```bin``` directory

**Comment by SlavomirMazurPantheon** [comment]
I would prefer to not log individual scripts as `extract_ietf_modules`, `gather_ietf_dependent_modules` ...., but log only start/end of `cronjob-daily` script and these individual scripts can be added as Additional information, so it will look more like this 
Image Pasted at 2023-1-13 08-44

**Comment by Fagtoy** [comment]
> I would prefer to not log individual scripts as `extract_ietf_modules`, `gather_ietf_dependent_modules` ...., but log only start/end of `cronjob-daily` script and these individual scripts can be added as Additional information, so it will look more like this Image Pasted at 2023-1-13 08-44

done
Screenshot from 2023-01-13 18-58-57
Screenshot from 2023-01-13 19-01-29

File: check_archived_drafts.py [hunk 1]
```diff
@@ -82,28 +81,24 @@ def __init__(
         self.missing_modules: list[str] = []
         self.incorrect_revision_modules: list[str] = []
 
-    def start_process(self):
-        start = int(time.time())
+    @job_log_decorator(file_basename=file_basename)
```
Comment (SlavomirMazurPantheon) [inline check_archived_drafts.py:84]: No need to use word `decorator` just name it `job_log`

**Review by SlavomirMazurPantheon — APPROVED** [review]

**Pull request merged**